### ISSUE TITLE
Looking for advice how to do a self-supervised post-training based on iBOT

### ISSUE THREAD

**Issue by steve-zeyu-zhang** [bytedance/ibot#38]
Hi team,

I'm looking for advice how to do a self-supervised post-training based on the ViT-B-16 pretraining of iBOT. e.g. Assume I have 800K unlabeled images, which are not appear in ImageNet, without any label, and I want to adapt the ImageNet pretraining to the 800K unlabeled data.

What should I pay attention to, e.g. epoch, warmup epoch, learning rate, teacher_temp, warmup_teacher_temp_epochs, etc.

I've tried a couple of times previously using the default configuration and epoch in the paper, e.g. teacher temp = 0.07, warmup_teacher_temp_epochs = 30, warmup_epochs = 10, lr (bs = 256) = 0.0005, lr (bs = 1024) =0.002,

But the loss was always decreasing a little and up to 12-13 then nan (stop training)

Any suggestions?

**Comment by justinkay** [comment]
Hello @steve-zeyu-zhang, did you ever find some settings that worked for you?
